docs/section11/Rule11-6.md

```diff
@@ -0,0 +1,35 @@
+# Service Water Heating - Rule 11-6
+
+**Schema Version:** 0.0.37  
+**Mandatory Rule:** True  
+**Rule ID:** 11-6  
+**Rule Description:** Piping losses shall not be modeled. 
```

Comment:
I think we should have a rule that checks this for the baselien and another that checks the proposed

Reply:
This is covered in the global baseline versus proposed check

Reply:
This needs to be changed to true if you want this to be covered by rule 1-6.

https://github.com/open229/ruleset-model-description-schema/blob/develop/schema-source/ASHRAE229_extra.schema.yaml#L5768

Reply:
it's covered here:  lines 101-103

just updated to reflect change to `are_thermal_losses_modeled`

[docs/section11/SHW comparison fields.yaml](url)